src/classes/STG_InstallScript_TEST.cls

```diff
@@ -32,7 +32,8 @@
 * @date 2014
 * @group Settings
 * @description Test Class for the Install Script for the NPSP package. 
-*/@isTest
+*/
+@isTest
```

Comment:
if you want them to filter by the testMethod pseudo-scope, yes.  There
still is the outstanding issue to detect @isTest, but I don't see that
happening anytime soon.  that said, this is lower in priority (in my
opinion) vs getting the descriptions in there.

Dave


David Habib | Senior Force.com Developer | salesforcefoundation.org

On Mon, Jan 26, 2015 at 9:11 AM, Nicolas Campbell <notifications@github.com>
wrote:

> In src/classes/STG_InstallScript_TEST.cls
> <https://github.com/SalesforceFoundation/Cumulus/pull/1282#discussion_r23546133>
> :
>
> > @@ -32,7 +32,8 @@
> >  * @date 2014
> >  * @group Settings
> >  * @description Test Class for the Install Script for the NPSP package.
> > -*/@isTest
> > +*/
> > +@isTest
>
> Do we have to change our test methods to use the testMethod annotation
> instead of @isTest <https://github.com/isTest> for apexDocs purposes?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/SalesforceFoundation/Cumulus/pull/1282/files#r23546133>
> .
>